### PR TITLE
Updated the about page text

### LINKED ISSUE #519: Update about page copy
updated text: https://docs.google.com/document/d/1HUivXOfS1bovVAJX2u_2QERqUSlU9dG2uQiC0vVIqDA/edit?pli=1

#### Acceptance Criteria:

- [ ] text changes 
- [ ] fix the link: [Find courses with Instructor Insights](https://ocwnext.odl.mit.edu/search/?f=Instructor%20Insights&s=-runs.best_start_date)
- [ ] update the "Dean's message" video to https://www.youtube.com/watch?v=1vSCB9V-Yqw
- [ ] add Angela Pignatiello, Administrative Assistant to the OCW team  (until I locate a picture, you can use the same grey box as the one for Phil Sandy)
- [ ] update the faculty advisory committee members

#### Out of Scope

- someday we might want to make this page editable through studio (esp. the faculty advisory team), but we don't have time to do it now.

### LINKED ISSUE #524: Home: remove all "coming soon" pop-ups
We should have working hrefs for all links on the home page, about page and educator page

#### Acceptance Criteria:

- [ ] "Get Started" on the home page should link to ZenDesk, https://mitocw.zendesk.com/hc/en-us/categories/4414679122971-Get-Started-with-MIT-OpenCourseWare

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
- [x] Testing
 - [x] Changes have been manually tested

#### What are the relevant tickets?
Closes #519, Closes #524

#### What's this PR do?
Accomplishes the following
- [x] text changes 
- [x] fix the link: [Find courses with Instructor Insights](https://ocwnext.odl.mit.edu/search/?f=Instructor%20Insights&s=-runs.best_start_date)
- [x] update the "Dean's message" video to https://www.youtube.com/watch?v=1vSCB9V-Yqw
- [x] add Angela Pignatiello, Administrative Assistant to the OCW team  (until I locate a picture, you can use the same grey box as the one for Phil Sandy)
- [x] update the faculty advisory committee members

#### How should this be manually tested?
Verify all required changes mentions in issue #519 have been made.

#### Screenshots (if appropriate)
Screenshot 2022-03-15 at 8 04 01 PM
Screenshot 2022-03-15 at 8 04 11 PM
Screenshot 2022-03-15 at 8 04 22 PM

### CODE REVIEW

File: www/content/milestones/2021.md [hunk 1]
```diff
@@ -3,6 +3,6 @@ title: 2021
 image: /images/about/timeline-image-1.png
 headless: true
 milestones:
-  - OCW celebrates two decades of open sharing
-  - "New platform launches: mobile responsive, enhanced search"
+  - OCW celebrates its 20th anniversary, two decades of open sharing (video)
```
Comment (pdpinch) [inline www/content/milestones/2021.md:6]: I think this should link to https://www.youtube.com/watch?v=8O2EPwUtWyY

**Review by MAbdurrehman12 — CHANGES_REQUESTED** [review]
Looks good. 
Noticed few things: I think, this might be something which can to be catered here.

1- Redbox: 2nd paragrapgh: 

Doc:
image

Site: 
image

2- Milestones > 2001-2005: Document has 5 points where as site has 7 points, Do we have to match it with the document? As there's this note in the document: `** Note: text has generally been rewritten throughout the page.**` 
Same goes for 2006-2010, 2011-2015, 2016-2020: Number/Count of bullet points are not matching with the document.

**Review by pdpinch — CHANGES_REQUESTED** [review]
Please add / change the following images:

Brett Paci

Curt Newton

Sarah Hansen


Yunpeng Wang



IMG_7579

**Comment by Wassaf-Shahzad** [comment]
> Please add / change the following images:
> 
> Brett Paci
> 
> Curt Newton
> 
> Sarah Hansen
> 
> Yunpeng Wang
> 
> IMG_7579

@pdpinch Can you also provide some meta data as well in regards to where to add them, their name and title.

**Comment by Wassaf-Shahzad** [comment]
> Looks good. Noticed few things: I think, this might be something which can to be catered here.
> 
> 1- Redbox: 2nd paragrapgh:
> 
> Doc: image
> 
> Site: image
> 
> 2- Milestones > 2001-2005: Document has 5 points where as site has 7 points, Do we have to match it with the document? As there's this note in the document: `** Note: text has generally been rewritten throughout the page.**` Same goes for 2006-2010, 2011-2015, 2016-2020: Number/Count of bullet points are not matching with the document.

@pdpinch @MAbdurrehman12, I just updated the text of the points in milestones  and did not remove any point. As the doc did not explicitly say to remove any point.
I just need some confirmation if I can remove  points in milestones which are not mentioned in document

**Comment by pdpinch** [comment]
> I just need some confirmation if I can remove points in milestones which are not mentioned in document

Yes, please remove the points that don't appear in the updated doc.

**Comment by pdpinch** [comment]
@Wassaf-Shahzad here's the issue for the home page link. I think you said you wanted to add it to this PR.

**Comment by pdpinch** [comment]
The new images are all people who are already on the page:

- [ ] [Brett Paci](https://user-images.githubusercontent.com/430126/158588147-e1eec8cc-e8bf-41a0-b67b-ec45e8019077.jpg)
- [ ] [Curt Newton](https://user-images.githubusercontent.com/430126/158588275-bee188cc-fee2-408b-9e56-1e5fa18b2186.jpg)
- [ ] [Sarah Hansen](https://user-images.githubusercontent.com/430126/158588305-ce39a7ea-bc7e-4401-b22c-9034fd60d7f8.jpeg)
- [ ] [Yunpeng Wang](https://user-images.githubusercontent.com/430126/158588411-9e3ae14f-5a99-4513-944f-d4f036c7802e.jpg)
- [ ] [Angela Pignatiello](https://user-images.githubusercontent.com/430126/158588451-f779e10a-9761-46a0-b1eb-549ebd01433a.jpg)

**Comment by Wassaf-Shahzad** [comment]
> @Wassaf-Shahzad here's the issue for the home page link. I think you said you wanted to add it to this PR.

Also added changes needed for issue #524

File: www/content/milestones/2016-2020.md [hunk 2]
```diff
@@ -3,11 +3,10 @@ title: 2016-2020
 image: /images/about/timeline-image-2.png
 headless: true
 milestones:
-  - There are more than 2,500 OCW courses and resources on the website, and over 6,000 videos on OCW’s YouTube channel, with 500 million visits and 5 billion minutes of learning.
-  - With over 2.5 million subscribers, OCW is the largest .edu channel on YouTube.
+  - OCW becomes the most-subscribed .edu channel on YouTube.
```
Comment (pdpinch) [inline www/content/milestones/2016-2020.md:6]: add link to https://www.youtube.com/c/mitocw

```suggestion
  - OCW becomes the most-subscribed .edu channel on [YouTube](https://www.youtube.com/c/mitocw).
```

File: www/content/milestones/2016-2020.md [hunk 3]
```diff
@@ -3,11 +3,10 @@ title: 2016-2020
...
   - The 200th Instructor Insights video is published through the OCW Educator program.
   - Interactive, searchable transcripts are added to videos.
```
Comment (pdpinch) [inline www/content/milestones/2016-2020.md:8]: remove these two lines
```suggestion
```

File: www/content/milestones/2016-2020.md [hunk 4]
```diff
@@ -3,11 +3,10 @@ title: 2016-2020
...
   - New MIT Open Learning Library launches in 2019, offering new, self-paced learning options complementing OCW and MITx.
```
Comment (pdpinch) [inline www/content/milestones/2016-2020.md:9]: link to https://openlearning.mit.edu/courses-programs/open-learning-library
```suggestion
  - New [MIT Open Learning Library](https://openlearning.mit.edu/courses-programs/open-learning-library) launches in 2019, offering new, self-paced learning options complementing OCW and MITx.
```

File: www/content/milestones/2016-2020.md [hunk 5]
```diff
@@ -3,11 +3,10 @@ title: 2016-2020
...
   - Chalk Radio podcast launches in February 2020.
```
Comment (pdpinch) [inline www/content/milestones/2016-2020.md:10]: add link
```suggestion
  - [Chalk Radio podcast](https://chalk-radio.simplecast.com/) launches in February 2020.
```

File: www/content/milestones/2016-2020.md [hunk 6]
```diff
@@ -3,11 +3,10 @@ title: 2016-2020
...
-  - The world turns to OCW during the COVID-19 pandemic, with traffic to the OCW website nearly doubling in April 2020.
+  - OCW website traffic nearly doubles as the COVID-19 pandemic disrupts life around the world.
```
Comment (pdpinch) [inline www/content/milestones/2016-2020.md:11]:
```suggestion
  - OCW supports learners around the world as the COVID-19 pandemic disrupts education.
```

File: www/content/milestones/2021.md [hunk 7]
```diff
@@ -3,6 +3,6 @@ title: 2021
 image: /images/about/timeline-image-1.png
```
Comment (pdpinch) [inline www/content/milestones/2021.md:3]: the title should be `2021+`

File: www/content/milestones/2021.md [hunk 8]
```diff
@@ -3,6 +3,6 @@ title: 2021
...
 headless: true
 milestones:
-  - OCW celebrates two decades of open sharing
-  - "New platform launches: mobile responsive, enhanced search"
+  - OCW celebrates its 20th anniversary, two decades of open sharing ([video](https://www.youtube.com/watch?v=8O2EPwUtWyY))
+  - "New OCW platform launches: mobile responsive, enhanced search and more"
```
Comment (pdpinch) [inline www/content/milestones/2021.md:7]:
```suggestion
  - OCW celebrates its 20th anniversary and two decades of open sharing ([video](https://www.youtube.com/watch?v=8O2EPwUtWyY)).
  - New OCW platform launches: mobile responsive, enhanced search and more
  - 440 OCW mirror sites around the world.
```

File: www/layouts/about/section.html [hunk 9]
```diff
@@ -315,59 +308,57 @@ <h1 class="text-center w-100 pb-6">OCW Milestones</h1>
...
-              Insights” section, which shares the instructors' perspectives on
-              topics of interest to practicing teachers, such as course design,
-              active learning, and engaging learners.
+              Many OCW courses include an “Instructor Insights” section for 
+              practicing teachers, covering topics such as course design, active 
+              learning methods, and strategies to engage learners.
             </p>
-            <a class="red-btn" href="/search/?type=Instructor%20Insights"
+            <a class="red-btn" href="/search/?type=Instructor%20Insights&s=-runs.best_start_date"
               >Find course with Instructor Insights</a
```
Comment (pdpinch) [inline www/layouts/about/section.html:331]:
```suggestion
              >Find courses with Instructor Insights</a
```

**Review by pdpinch — CHANGES_REQUESTED** [review]
another round of text edits, based on the changes in https://docs.google.com/document/d/1HUivXOfS1bovVAJX2u_2QERqUSlU9dG2uQiC0vVIqDA/edit?pli=1

When the PR is updated I'll ask them to review on netlify again

File: www/content/milestones/2001-2005.md [hunk 10]
```diff
@@ -4,11 +4,8 @@ image: /images/about/timeline-image-1.png
...
-  - The first 50 courses are published in 2002. [(View on Internet Archive)](https://web.archive.org/web/20021023205659/http:/ocw.mit.edu/index.html)
-  - OCW officially launches in 2003 with 500 courses. [(View on Internet Archive)](https://web.archive.org/web/20031008143931/http:/ocw.mit.edu/index.html)
-  - OCW adopts a [Creative Commons license](https://creativecommons.org/licenses/by-nc-sa/4.0/) in 2004.
-  - OpenCourseWare Consortium forms in 2005 (renamed the Open Education Consortium in 2014, and [OE Global Consortium](https://www.oeconsortium.org/) in 2020).
+  - Pilot site of 50 courses [(View on Internet Archive)](https://web.archive.org/web/20021023205659/http:/ocw.mit.edu/index.html) followed by 2003 official website launch with 500 courses. [(View on Internet Archive)](https://web.archive.org/web/20031008143931/http:/ocw.mit.edu/index.html)
+  - OCW adopts a [Creative Commons license](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+  - OpenCourseWare Consortium forms (now called [OE Global Consortium](https://www.oeconsortium.org/)).
   - The first [OCW mirror site](https://ocw.mit.edu/about/mirror-site-program/) is established in Africa.
```
[the remark this answers is not in the document]
Comment (Wassaf-Shahzad) [reply www/content/milestones/2001-2005.md:10]: Which link ?

**Comment by Wassaf-Shahzad** [comment]
Updated text and also included changes for issue #530

File: www/content/milestones/2001-2005.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (pdpinch) [reply www/content/milestones/2001-2005.md:10]:
```suggestion
  - The first OCW mirror site is established in Africa.
```

**Review by pdpinch — CHANGES_REQUESTED** [review]
Thanks. I know that was a lot of small text changes, but it looks like you missed a couple:

- [ ] https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829622620
- [ ] https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829622412
- [ ] https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829621858
- [ ] https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829618940

**Comment by Wassaf-Shahzad** [comment]
> Thanks. I know that was a lot of small text changes, but it looks like you missed a couple:
> 
> * [ ]  https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829622620
> * [ ]  https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829622412
> * [ ]  https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829621858
> * [ ]  https://github.com/mitodl/ocw-hugo-themes/pull/522/files#r829618940

Ohh sorry, for some reason, these changes were showing up as `Hidden conversation` for me. So I missed them.
Addressed them as well

File: www/layouts/about/section.html [hunk 11]
```diff
@@ -173,17 +173,17 @@ <h1 class="text-center-mobile">
...
+            keep growing and evolving.
             <br />
             <br />
-            We're excited to introduce our new platform - which you are using
-            now - to bring smartphone users into OCW's global community…provide
-            more powerful and insightful ways to discover just the content you
-            need… and be a foundation for new features and learning experiences
-            for years to come.
+            Our new platform, which you are now using, brings smartphone 
+            users fully into OCW's global community… provides more powerful
```
Comment (pdpinch) [inline www/layouts/about/section.html:183]: sorry, I mistakenly added a space here after the `...`. please remove. 
```suggestion
            users fully into OCW's global community…provides more powerful 
```

**Review by pdpinch — CHANGES_REQUESTED** [review]
one more tiny change

**Review by pdpinch — APPROVED** [review]
The copy is good. Let's merge this. 

There may be updates to the images later, but this is good enough for release. 

Thank you for your patience!

**Pull request merged**